Code-Sleep-Python/text-to-speech/tts.py

```diff
@@ -1,5 +1,5 @@
 from gtts import gTTS
 
-text = ("Hello world!")
+text = input("Enter the text you want to convert: ")
```

Comment:
I guess this might break the functionality.
Have you tested this?

Also it would be good if you can write a test case for this

Reply:
I don't actually understand how it breaks functionality. When I tested, it was able to convert the text I entered to an audio file, which is what it is supposed to do. Am I missing something here?

Reply:
In some cases, input(text) does not always work, depending on the version. I have tested it locally in this case. And it works fine.